domains/test/service/src/domain.rs

```diff
@@ -221,6 +221,11 @@ where
             .xdm_gossip_worker_builder()
             .gossip_msg_sink();
 
+        let mut maybe_operator_id = None;
+        if role.is_authority() {
+            maybe_operator_id = Some(0)
+        }
```

Comment:
Non-mutable should be preferred:
```suggestion
        let maybe_operator_id = if role.is_authority() {
            maybe_operator_id = Some(0)
        } else {
            None
        };
```
More functional is also possible and is shorter:
```suggestion
        let maybe_operator_id = role.is_authority().then_some(0);
```